versioned_docs/version-v4.0.0/token/token-wrapping.md

```
@@ -0,0 +1,74 @@
+
+   ![Staking hub wrapper](/img/token/wrapper-2.jpg)
+
+## 4. Enter the amount to wrap
+
+   In the "**xHOPR**" field, enter the amount of HOPR tokens you want to wrap into wxHOPR.
+
+   ![Staking hub wrapper](/img/token/wrapper-3.jpg)
+
+## 5. Confirm the transaction
+
+   After entering the desired amount, click "**Swap**" to initiate the wrapping process. Confirm the transaction in your wallet when prompted. Once the transaction is confirmed, you can import the wrapped HOPR token (**wxHOPR**) into your wallet by using the correct smart contract address, which can be found [here](./acquiring-hopr-tokens.md#hopr-tokens-on-different-chains).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)